catala/aixecar/aixecar.md

```diff
@@ -1,3 +1,8 @@
 T'aixeques del llit i recorres l'habitació intentant recordar on ets
 
-quan de sobte els mobles començen a convertir-se en xocolata.
\ No newline at end of file
+quan de sobte els mobles començen a convertir-se en xocolata.
+
+Què fas?
+
+[Et menges un moble](menjar/menjar.md)
+[Corres al llit i t'amagues sota els llençols](amagar-se/amagar-se.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.